chisel/src/main.rs

```diff
@@ -327,12 +325,13 @@ fn chisel_execute(context: &ChiselContext) -> Result<bool, &'static str> {
 
             // If the module was mutated, serialize to file.
             if original != module {
+                let serialized = module.to_bytes().unwrap();
                 if let Some(path) = context.outfile() {
                     chisel_debug!(1, "Writing to file: {}", path);
-                    serialize_to_file(path, module).unwrap();
+                    write(path, serialized).unwrap();
```

Comment:
Can we use `expect` with a little description that the write operation failed for some reason?

Reply:
We could but your previous wasn't doing that either and I expected the big CLI PR will deal with it so wanted to minise the change here.